src/Controller/ModulesController.php

```diff
@@ -267,7 +268,7 @@ public function save() : ?Response
 
             // save data
             $response = $this->apiClient->save($this->objectType, $requestData);
-        } catch (BEditaClientException $e) {
+        } catch (\RuntimeException | BEditaClientException | UploadException $e) {
             // Error! Back to object view or index.
             $this->log($e, LogLevel::ERROR);
             $this->Flash->error($e, ['params' => $e->getAttributes()]);
```

Comment:
From Scrutinizer: ```The method getAttributes() does not exist on RuntimeException. It seems like you code against a sub-type of RuntimeException such as PhpParser\Error or BEdita\SDK\BEditaClientException or Cake\Core\Exception\Exception```
We probably could use `\Cake\Http\Exception\InternalErrorException` instead of `\RuntimeException`